src/user/views/SignUpPage.vue

```diff
@@ -0,0 +1,286 @@
+<template>
+  <main class="signup-page">
+    <section class="marketing">
+      <div class="marketing--content">
+        <div class="marketing--heading">
+          <h2>The Only Tool You Need</h2>
+          <h2>for Group Scheduling</h2>
+        </div>
+
+        <ul class="marketing--list">
+          <li class="marketing--item">
+            <svg
+              width="60"
+              height="60"
+              viewBox="0 0 60 60"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M30 0C13.455 0 0 13.455 0 30C0 46.545 13.455 60 30 60C46.542 60 60 46.545 60 30C60 13.455 46.542 0 30 0ZM33 42V30L15 42V18L33 30V18L51 30L33 42Z"
+                fill="var(--color-card)"
+              />
+            </svg>
+            <h4>Faster event creation</h4>
+          </li>
+          <li class="marketing--item">
+            <svg
+              width="60"
+              height="60"
+              viewBox="0 0 60 60"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M30 0C13.737 0 0 13.737 0 30C0 46.263 13.737 60 30 60C46.263 60 60 46.263 60 30C60 13.737 46.263 0 30 0ZM30 15C35.181 15 39 18.816 39 24C39 29.184 35.181 33 30 33C24.822 33 21 29.184 21 24C21 18.816 24.822 15 30 15ZM14.682 44.316C17.373 40.356 21.861 37.716 27 37.716H33C38.142 37.716 42.627 40.356 45.318 44.316C41.484 48.42 36.045 51 30 51C23.955 51 18.516 48.42 14.682 44.316Z"
+                fill="var(--color-card)"
+              />
+            </svg>
```

Comment:
Can you refactor this as an icon under the `AppIcon` component? See the files in the `/common/app-icon` folder for reference.

Reply:
Done. I had intended that but totally forgot